test/e2e/setup_test.go

```diff
@@ -37,6 +39,12 @@ func (s *SetupTests) Run(t *testing.T) {
 	require.True(t, running, "The Kluster must be Running")
 }
 
+func (s *SetupTests) CreateSecurityGroup(t *testing.T) {
+	sg, err := securitygroups.Create(s.OpenStack.Network, securitygroups.CreateOpts{Name: "kubernikus"}).Extract()
```

Comment:
I would create a dedicated security group that is named after the current soak test cluster.